nx_parallel/algorithms/cluster.py

```diff
@@ -47,15 +47,16 @@ def _compute_clustering_chunk(node_iter_chunk):
     else:
         node_iter = list(G.nbunch_iter(nodes))
 
-    total_cores = nxp.cpu_count()
+    n_jobs = nxp.cpu_count()
+    n_jobs = 1 if n_jobs is None else n_jobs  # changing the default
```

Comment:
Let's try putting this into the central function as you described in the zoom call.